v1.14.0

kenhys opened the pull request:
**Which issue(s) this PR fixes**: 

Fixes N/A

**What this PR does / why we need it**: 

prepare v1.14.0 changelog

**Docs Changes**:

N/A

**Release Note**: 

N/A

ashie commented:
FYI: We've released 1.14.0.rc: https://rubygems.org/gems/fluentd/versions/1.14.0.rc

ashie commented on CHANGELOG.md:
```diff
@@ -1,3 +1,42 @@
+### Enhancement
+
+* Added `enable_input_metrics`, `enable_size_metrics` system configuration parameter
+  This feature need to pay high CPU cost, so input event metrics
```
It thinks that it's too exaggerated. Probably CPU cost isn't so increased in most cases.
```diff
- This feature need to pay high CPU cost
+ This feature might need to pay higher CPU cost
```

kenhys replied:
Thanks, I've fixed it.

kenhys commented:
Add #3481 entry.

kenhys commented:
changelog and documentation status

* Added `enable_input_metrics`, `enable_size_metrics` system configuration parameter
  `--enable-input-metrics`,`--enable-size-metrics` command line
* [x] Added reserved word `@ROOT` for getting root router.
  * https://github.com/fluent/fluentd-docs-gitbook/pull/356
* [x] in_syslog: Added `send_keepalive_packet` option
  * https://github.com/fluent/fluentd-docs-gitbook/pull/354/
* [x] in_http: Added `cors_allow_credentials` option.
  * https://github.com/fluent/fluentd-docs-gitbook/pull/355
* [x] Added metrics plugin mechanism
  * https://github.com/fluent/fluentd-docs-gitbook/pull/347/

kenhys commented:
Now ready to ship it!

cosmo0920 commented:
:tada:

cosmo0920 commented:
@kenhys @ashie Can we release fluentd x86-mingw32 and x64-mingw32 version?
They are not released and it causes CI error on https://github.com/calyptia/fluent-plugin-metrics-cmetrics/runs/3457273532#step:5:1

kenhys commented:
Done!

cosmo0920 commented:
Thanks!
